packages/block-editor/src/components/inserter/index.native.js

```diff
@@ -315,13 +319,18 @@ export default compose( [
 			? selectedBlockIndex + 1
 			: endOfRootIndex;
 
+		const blockDoNotSupportInserter =
+			! isAppender &&
+			getBlockSupport( getBlockName( end ), 'inserter' ) === false;
```

Comment:
>Actually we wouldn't want to copy and duplicate, we just want to override a single attribute.

Yeah thats true, we only need to import `index.js` content and override support inserter setting it to true

```jsx
supports: {
		inserter: true,
         }
```

>Not sure I understand correctly, in that case the value would default to true, which is something we don't want?

Actually we want to be able add Column with Inserter but mentioned PR block us - that's why we provide walk-around to disable it until it gets merge. However I had impression that we do not want to modify `index.js` at all as well as overriding its prop in proposed way. That's why all those changes in Inserter was made.

If we can temporary override support inserter to true we can do not touch the code in inserter at all. 

We must override that to avoid being affected after mentioned PR gets merge so why not override it with `true` value and achieve the result that we want to achieve immediately ?

Reply:
I think in this case we should override because apparently we want to keep the UX this way.

Reply:
> I think in this case we should override because apparently we want to keep the UX this way.

Just to make sure:
option1: override support: inserter to `false` and keep changes in Inserter to have it disable in Column block until support inserter PR gets merged

or

option2: override support: inserter to `true` and remove changes in Inserter to have it enable in Column block

Reply:
Option 2 sounds good. Just to be more specific, what I have in mind is:

Set `inserter:true` in index.native.js, merge this PR 
Wait for [web side PR](https://github.com/WordPress/gutenberg/pull/20502/files) to get merged
Remove index.native.js

Reply:
done